packages/block-editor/src/components/block-tools/style.scss

```diff
@@ -59,7 +59,6 @@
 
 // Sibling inserter / "inbetweenserter".
 .block-editor-block-list__empty-block-inserter,
-.block-editor-default-block-appender,
```

Comment:
The default inserter will not be used outside of the iframe editor, so I removed it.